docker-compose.yml

```diff
@@ -1,9 +1,12 @@
-version: '3.8'
+# `version` key here is optional and no longer required in modern docker-compose versions.
+# version: '3.8'
```

Comment:
I agree. We should get rid of this completely then :)

Reply:
I've seen issues a few times when the docker versions are too old on machines when migrating or first running. If it's there in the code it's explicit for people who hit problems.

Reply:
You mean "docker-compose" versions? The docker daemon itself does not care about docker-compose.yml at all.

If you could provide any example where a missing version can lead to problems, this would be great. As I don't think that's a problem at all, I am open to learn something new 👍

fyi: See also my comment [here](https://github.com/docker-mailserver/docker-mailserver/issues/2254#issuecomment-946695253)

Reply:
What happens when you use 3.0 options with a version that doesn't support them? That's what my experience has been.

Reply:
I guess docker-compose will fail to execute. But a version information on top will not prevent that?

> Top-level version property is defined by the specification for backward compatibility but is only informative.

PS: We don't use any version specific options.

Reply:
Yep, it's more of just for people who do experience issues to know what we require. If there aren't any 3.X specific options we're good though.